crossing-training.lic

```diff
@@ -738,11 +738,17 @@ class CrossingTraining
   end
 
   def train_stealth
-    hide_in 851
-    hide_in 850
-    hide_in 764
-    hide_in 5992
-    hide_in 992
+    if DRSkill.getrank('Stealth') >= 25
+      echo('Skill too high to learn this way, removing Stealth from training')
+      @settings.crossing_training.delete('Stealth')
+      nil
```

Comment:
hmm this was return but rubocop changed it to nil

Reply:
Since ruby returns the last thing evaluated, `return` and `nil` would be the same